engine/src/main/java/io/zeebe/engine/processing/incident/ResolveIncidentProcessor.java

```diff
@@ -102,7 +94,8 @@ private void attemptToResolveIncident(
     final boolean isJobIncident = jobKey > 0;
 
     if (isJobIncident) {
-      attemptToSolveJobIncident(jobKey, streamWriter);
+      // todo (#6000): resolve incident for UNHANDLED_ERROR_EVENT
+      // at time of writing the workflow cannot be fixed, so this incident cannot be resolved
```

Comment:
Maybe we add here a log statement?

Reply:
Good idea

Reply:
It now rejects the resolve command. No need to log anything. Just inform the client.